### ISSUE TITLE
Can you share your loss graph while training the syncnet?

### ISSUE THREAD

**Issue by SeongYeonPark** [Rudrabha/Wav2Lip#334]
I'm having a hard time trying to make the evaluation loss fall to ~0.25.
Maybe I don't have enough data, or maybe i should watch the training go a little further.

Either way, your version of the loss graph (while training the syncnet) would be very helpful for me to gain some intuition on this matter.

Thank you.

**Comment by SeongYeonPark** [comment]
![syncnet_lossgraph](https://user-images.githubusercontent.com/70704127/143531026-f1c6e6fa-7397-449a-bf08-a6424b22251f.png)
(By the way, the x axis is epochs. For the dataset i used, 1epoch was about 1.3k steps. (batch size = 64))
My current training graph looks like that. I've passed the 0.69 loss 'barrier', but the loss shows overfitting around an eval loss of 0.55.

**Comment by prajwalkr** [comment]
> I'm having a hard time trying to make the evaluation loss fall to ~0.25. Maybe I don't have enough data, or maybe i should watch the training go a little further.


The loss value we mentioned is only for training on the LRS2 train set. Other datasets need preprocessing such as sync correction etc., to work reliably
